changelog/unreleased/features/1749--vast-dev-docker-image.md

```diff
@@ -0,0 +1,5 @@
+We now offer a [`tenzir/vast-dev`](https://hub.docker.com/r/tenzir/vast-dev)
+variant besides the regular
+[`tenzir/vast`](https://hub.docker.com/r/tenzir/vast) Docker images that is
+designed to be used in development contexts, e.g., when building additional
+plugins.
```

Comment:
It would be nice to have a quick not on how they differ, something more about the image than just the (plugin) use case.